### ISSUE TITLE
FreeCad 1.0 support

### ISSUE THREAD

**Issue by Mathias-Boulay** [dracula/freecad#49]
Hello maintainers, with the freecad 1.0 release, it seems like the theme is outdated in quite a few ways.

I attached a few pictures for you to get an idea, I have a feeling that there is much more to fix.

_random red buttons_
![image](https://github.com/user-attachments/assets/860d9a4d-23e2-47b7-8d37-7c0a098d3540)

_improper padding on a few elements_
![image](https://github.com/user-attachments/assets/728e5947-989b-4b47-b0f9-6aa7ef6ab08d)

_icons on dropdown being cut to the right, dropdown arrow having a different (and wrong ?) color from the rest of said dropdown_
![image](https://github.com/user-attachments/assets/6145988d-badf-4d22-84e5-fe9d0f7769bb)

_More read buttons, large padding where it isn't supposed to be_
![image](https://github.com/user-attachments/assets/aaf76d25-8469-44b7-9f64-e3efd5c5175b)

Edit 1:

_wrong padding_
![image](https://github.com/user-attachments/assets/bf268d6b-6557-4812-8ee7-f4012e2e6ab7)

**Comment by Chrismettal** [comment]
Yep, noticed during RC phase but didn't yet get around to updating. Now that 1.0 is fully released we can start working on the conflicts soon. Currently still out of office, planning to work on this and the other Dracula themes I maintain over Christmas.

**Comment by MaZe3D** [comment]
I hope this is the right place for this but I cannot read Error-messages with this theme. They have basically no contrast on a white background
![grafik](https://github.com/user-attachments/assets/983ebf55-d729-4e4e-8613-05e2250b7a7c)

**Comment by Mathias-Boulay** [comment]
Heads up since Christmas has passed, I am still available to test whenever the theme gets updated 👍

**Comment by Chrismettal** [comment]
Random red buttons fixed with a21c989:

![Image](https://github.com/user-attachments/assets/f4b23de5-8ee7-4b05-8332-1e63a4e06a9f)

**Comment by Chrismettal** [comment]
QToolBar "Dropdowns" somewhat improved with :

![Image](https://github.com/user-attachments/assets/8e78bc5e-cc7a-44f3-9061-eee663864810)

**Comment by Mathias-Boulay** [comment]
I've been testing it while making a small replacement piece for my chair. Theme wise, it seems to be much better than last time.

**Comment by brandonbeaumia** [comment]
All 1.0 issues reported here fixed between #52, #54, #55, and commit a21c989. Upped to 0.0.9, please update and continue reporting any new issues individually.
